src/resources.js

```diff
@@ -40,6 +38,7 @@ class Resource {
     var _results = results || [];
 
     this.signatureFactory.setUrl(this.url);
+    this.signatureFactory.setHeaders({'user-agent': 'Usabilla API Client for Node.js'});
```

Comment:
Good be nice to add the version of the client (which you could grab from the package.json).

Maybe something like `Usabilla API Node Client/1.0.0-beta.1`